modules/yieldlabBidAdapter.js

```diff
@@ -174,6 +174,37 @@ export const spec = {
       }
     })
     return bidResponses
+  },
+
+  /**
+   * Register the user sync pixels which should be dropped after the auction.
+   *
+   * @param {SyncOptions} syncOptions Which user syncs are allowed?
+   * @param {ServerResponse[]} serverResponses List of server's responses.
+   * @param {Object} gdprConsent Is the GDPR Consent object wrapping gdprApplies {boolean} and consentString {string} attributes.
+   * @param {string} uspConsent Is the US Privacy Consent string.
+   * @return {UserSync[]} The user syncs which should be dropped.
+   */
+  getUserSyncs: function (syncOptions, serverResponses, gdprConsent, uspConsent) {
+    const syncs = [];
+
+    if (syncOptions.iframeEnabled) {
+      let params = [];
+      params.push(`ts=${timestamp()}`);
+      params.push(`type=h`)
+      if (gdprConsent && (typeof gdprConsent.gdprApplies === 'boolean')) {
+        params.push(`gdpr=${gdprConsent.gdprApplies ? '1' : '0'}`);
+      }
+      if (gdprConsent && (typeof gdprConsent.consentString === 'string')) {
+        params.push(`consent=${gdprConsent.consentString}`);
```

Comment:
Even though we are currently ignoring the `gdpr` parameter in adserver we could use `gdpr_consent` for the consent string parameter to be in line with pixie and have it uniform.
```suggestion
        params.push(`gdpr_consent=${gdprConsent.consentString}`);
```